const checkUser = async (req, res, next) => {
    try {
        console.log("Verifying user...");





    } catch (error) {

    }
}

export default checkUser;